### PR TITLE
AngularJS module

### PR DESCRIPTION
**Pull request by DavertMik**
* extends WebDriver module
* adds async waiter (taken from Protractor) after each action
* adds strict locator rule for finding elements by model
* added a test suite and basic test

### CODE REVIEW

File: src/Codeception/Module/AngularJS.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Codeception\Module;
+
+use Codeception\Step;
+use WebDriverBy;
```
Comment (DavertMik) [inline src/Codeception/Module/AngularJS.php]: should be `Facebook\Webdriver\WebDriverBy`

**Comment by DavertMik** [comment]
@zbateson @Naktibalda can you take a brief look into it?

**Comment by Naktibalda** [comment]
I can't comment about Javascript code and have nothing to say about PHP.

**Comment by DavertMik** [comment]
> I can't comment about Javascript code

just don't care of it, it was copy-pasted from Protractor clientscripts.js.
the only thing I'm not satisfied here are the tests, as they are launching 3rd-party Angular application. I hope soon we get our own test site with Angular

**Comment by DavertMik** [comment]
https://github.com/angular/protractor/blob/master/lib%2Fclientsidescripts.js#L50

**Pull request merged**